source/renderer/app/config/sidebarConfig.js

```diff
@@ -28,4 +34,10 @@ export const CATEGORIES = [
   CATEGORIES_BY_NAME.SETTINGS,
 ];
 
+export const CATEGORIES_FOR_STORYBOARD = [
```

Comment:
@yakovkaravelov app code should have no reference to storybook. Can we move this config to storybook?